pkg/kubelet/network/hostport/hostport_manager_test.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/stretchr/testify/assert"
 	"k8s.io/api/core/v1"
 	utiliptables "k8s.io/kubernetes/pkg/util/iptables"
+	utilexec "k8s.io/utils/exec"
```

Comment:
don't rename packages unless it is necessary

Reply:
This one is pretty common